regexes/device/mobiles.yml

```
@@ -621,11 +621,11 @@ Advance:
 
 # AGM
 AGM:
-  regex: 'AGM A8 SE'
+  regex: 'AGM ([^/;]+)(?: Build|\))'
```

You should use this:
```suggestion
  regex: 'AGM ([^/;\)]+)(?: Build|\))'
```
Otherwise it might match too much if the Useragent contains two `)` after the `AGM`